features/reservations/cancel_subscriber_voucher1.feature

```diff
@@ -0,0 +1,22 @@
+Feature: subscriber can make subscriber reservations
```

Comment:
can we give this feature file a more meaningful name than `cancel_subscriber_voucher1` ?  Names are important hints as to what something does.